interp/notation.ml

```diff
@@ -810,8 +814,9 @@ let find_arguments_scope r =
   with Not_found -> []
 
 let declare_ref_arguments_scope ref =
-  let t, _ = Global.type_of_global_in_context (Global.env ()) ref in
-  let (scs,cls as o) = compute_arguments_scope_full t in
+  let sigma, env = Evd.empty, Global.env () in (* FIXME? *)
+  let typ = EConstr.of_constr @@ fst @@ Global.type_of_global_in_context env ref in
+  let (scs,cls as o) = compute_arguments_scope_full sigma typ in
```

Comment:
Idem, I would pass `env` and `sigma` to `declare_ref_arguments_scope`, and further. Then, the day we want to support implicit arguments or scopes on goal variables (which could be useful with big proofs), this will almost be ready, even if the goal variable mentions an evar.

Or, alternatively, to replace the `FIXME` by a comment telling that terms with evars are not (yet) supported here.

Reply:
Done, but see the mess that we get on interp/declare.ml